Source/Core/Models/CardScheme.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CardScheme.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 05/08/2022.
+//
+
+public enum CardScheme {
```

Comment:
no need. please undo

Reply:
This is needed, have documented in PR description. Will copy here as well for ease of discovery:

> created CardScheme as a frames client interface for Card.Scheme. This enables end user to continue adding cases via nicer non associated value syntax for all cases. The parameter is also irrelevant to frames its only for internal handling so wouldn't make sense to expose it if we can avoid.

```Swift
// CardScheme
[.visa, .maestro, .mastercard]

// Card.Scheme
[visa, .maestro(), .mastercard]
```

Short version to why its needed:
**Consumer should not be concerned with variations of maestro, so the associated value should not be exposed**

Reply:
Ok makes sense. Is this now preferred to use across the SDK instead of `Card.Scheme`?

Reply:
It depends on use case, they solve different problems. I think internally Card.Scheme is still the preferred way as it exposes the complete scheme object functionality, whereas CardScheme is a prettier version for the public interface.